Assets/Scripts/SS3D/Systems/Health/Indefatigable.cs

```diff
@@ -0,0 +1,43 @@
+using UnityEngine;
+
+namespace SS3D.Systems.Health
+{
+    /// <summary>
+    /// Represents an entity that does not need stamina (robots etc). It will always have
+    /// sufficient stamina to perform any task where stamina is required.
+    /// </summary>
+    /// <inheritdoc cref="SS3D.Systems.Health.IStamina" />
+    public class Indefatigable : IStamina
```

Comment:
I'm not sure I like there being an interface and an extra class for infinite stamina. Could we not omit the stamina class instead (leave null)?

Reply:
Yep, I will remove the Indefatigable class and just manage with null. I would like to keep the IStamina interface though, it may come in helpful down the line.